src/Sulu/Bundle/AdminBundle/Resources/js/components/ImageRectangleSelection/ImageRectangleSelection.js

```diff
@@ -80,22 +79,47 @@ class ImageRectangleSelection extends React.Component<Props> {
 
     handleRectangleSelectionChange = (data: ?SelectionData) => {
         const {onChange} = this.props;
-        onChange(data ? this.rounding.normalize(this.scaledDataToNatural(data)) : undefined);
```

Comment:
@alexander-schranz This will fix a bug that makes this component fail to recognize the minimum size of pictures, but it also means that the result is not rounded anymore, e.g. decimal values are sent to the server. Do you think that this can cause problems?